ui/src/components/MainContainer.js

```diff
@@ -168,51 +173,54 @@ const MainContainer = () => {
         <PrivateRoute
           exact
           path="/clear-cache/"
-          permissions={['admin', 'stewardship']}
+          permissions={['t3ch', 'admin']}
           component={ClearCache}
         />
         <PrivateRoute
           exact
           path="/admin"
-          permissions={['admin', 'stewardship']}
+          permissions={['admin', 'stewardship', 't3ch']}
           component={AdminLoginContainer}
         />
-        {/* <PrivateRoute
-          exact
-          path="/admin/home"
-          permissions={["admin", "stewardship"]}
-          component={AdminHome}
-        />
-        <PrivateRoute
-          exact
-          path="/admin/users"
-          permissions={["admin", "stewardship"]}
-          component={AdminUser}
-        />
-        <PrivateRoute
-          exact
-          path="/admin/forms"
-          permissions={["admin", "stewardship"]}
-          component={AdminForm}
-        />
-        <PrivateRoute
-          exact
-          path="/admin/giving"
-          permissions={["stewardship"]}
-          component={AdminGiving}
-        />  */}
         <PrivateRoute
```

Comment:
why are there 2?

Reply:
cuz the logout will redirect to admin/logout instead of /admin, should I just change the logout.js then?